java/connecting-to-bigtable/README.md

```diff
@@ -0,0 +1,12 @@
+# Connecting to Cloud Bigtable
+
+This is a collection of samples relating to connecting to Cloud Bigtable.
+
+
+## Testing samples 
+
+To run tests:
+```
+export GCLOUD_PROJECT=test-project
+mvn test -Dbigtable.projectID=test-project -Dbigtable.instanceID=test-instance
```

Comment:
Env var + system property used here for project id. its worth simplifying and removing `GCLOUD_PROJECT` in this case.

Reply:
Done